target_chains/ethereum/examples/coin_flip/README.md

```diff
@@ -17,11 +17,10 @@ Next, choose a network to run the example on.
 The example has been deployed on the following networks:
 
 ```
-| Chain Name      | Address                                    | RPC                                        |
-|-----------------|--------------------------------------------|--------------------------------------------|
-| optimism-goerli | 0x3bA217Cd7840Cc5B34FD5B7263Cebd8CD8665788 | https://goerli.optimism.io                 |
-| avalanche-fuji  | 0xE7E52C85907d59C45b2C56EF32B78F514F8c547a | https://api.avax-test.network/ext/bc/C/rpc |
-| eos-evm-testnet | 0x413405Aee2db95cb028B60CBAd87FC0B932947f4 | https://api.testnet.evm.eosnetwork.com/    |
+| Chain Name       | Address                                    | RPC                                        |
+|------------------|--------------------------------------------|--------------------------------------------|
+| optimism-sepolia | 0x2eE67fF5d8548fF544f2c178a0FcAFe503A634Be | https://sepolia.optimism.io/               |
```

Comment:
It would be great if you can verify the contracts so they can see what's going on in the explorers too.